srcopsmetrics/entities/readme.py

```diff
@@ -41,22 +43,36 @@ def analyse(self) -> List[GithubContentFile]:
         if self.previous_knowledge is None or len(self.previous_knowledge) == 0:
             return [self.get_raw_github_data()]
 
-        if self.previous_knowledge["readme"]["size"] == self.repository.get_readme().size:
+        try:
+            readme = self.repository.get_readme()
+        except UnknownObjectException:
+            # TODO: consider storing ReadMe deletion events
+            return []
```

Comment:
isn't this logic the same as in `self.get_raw_github_data()`, why do we need that?

Reply:
absolutely true, resolved :vulcan_salute: